src/components/Book.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+
+const Book = ({ book, onDelete }) => (
+  <div>
+    <h3>{book.title}</h3>
+    <p>{book.author}</p>
+    <Button onClick={onDelete}>Delete</Button>
```

Comment:
- [ ] Let's kindly refer to the comment above :arrow_up: This button should include an event handler to remove a book from the books array with the id attribute and the removeBook action should be dispatch